package.json

```diff
@@ -30,11 +30,18 @@
     ]
   },
   "dependencies": {
+    "@graphql-tools/graphql-file-loader": "^7.3.3",
+    "@graphql-tools/load": "^7.4.1",
+    "@graphql-tools/schema": "^8.3.1",
+    "@graphql-tools/stitch": "^8.4.1",
+    "@graphql-tools/url-loader": "^7.5.2",
+    "@graphql-tools/wrap": "^8.3.1",
     "@typegoose/auto-increment": "^1.0.0",
     "@typegoose/typegoose": "^9.1.0",
     "apollo-server-core": "^3.4.0",
     "apollo-server-express": "^3.4.0",
     "class-validator": "^0.13.1",
+    "cross-undici-fetch": "^0.0.20",
```

Comment:
What is this package? Seems it's in a very early version, couldn't we use something more popular?

Reply:
Actually is no longer needed, removed